charts/matrix-server/templates/configmap-antiscamspamchecker.yaml

```diff
@@ -0,0 +1,289 @@
+apiVersion: v1
```

Comment:
I think we should only declare this configmap when antiscam is true, just adding `{{ if .Values.antiscam }}` at the beginning of this file (and `{{end}}` at the end) should be enough